File: src/main/java/org/sexedu/bot/utils/HTTP.java
package org.sexedu.bot.utils;

import cn.hutool.http.HttpUtil;

import java.util.Map;

public class HTTP {
    public static final String SERVER = "http://localhost:8080/";
    public static String sendPost(String url, Map<String, Object> paramMap) {
        return HttpUtil.post(SERVER + url, paramMap);
    }
}

File: src/main/java/org/sexedu/bot/utils/HTTP.kt
package org.sexedu.bot.utils

import cn.hutool.http.HttpUtil

object HTTP {
    const val SERVER = "http://localhost:6000/"
    fun sendPost(url: String, paramMap: HashMap<String, String>): String {
        return HttpUtil.post(SERVER + url, paramMap as Map<String, Any>?)
    }
}